doc/chomp_interface/scripts/collision_scene_example_sparse.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+import rospy
+from moveit_commander import RobotCommander, PlanningSceneInterface
+import geometry_msgs.msg
+import time
+
+class CollisionSceneExampleSparse(object):
```

Comment:
This is a ton of code duplication. You could achieve this in a single script that takes in a command line argument telling it which scene to load

Reply:
removed duplicate code and merged into single file and behavior now depends on command line argument ("cluttered" or "sparse")